src/Compiler/Checking/PatternMatchCompilation.fs

```diff
@@ -194,7 +194,15 @@ let ilFieldToTastConst lit =
     | ILFieldInit.Double f -> Const.Double f
 
 exception CannotRefute
-let RefuteDiscrimSet g m path discrims =
+
+[<Struct>]
+[<RequireQualifiedAccess>]
+type internal CounterExampleType = 
+    | EnumCoversKnown // maps to EnumMatchIncomplete exn
```

Comment:
Maybe move comments to `///` ones, so they are available in tooltips in other places?